spec/operators/expand-spec.ts

```diff
@@ -21,6 +22,19 @@ describe('Observable.prototype.expand', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
   });
 
+  it('should work with scheduler', () => {
+    const e1 =    hot('--x----|  ', {x: 1});
+    const e1subs =    '^        !';
+    const e2 =   cold('--c|      ', {c: 2});
```

Comment:
We could make this test a bit easier to read. `e2` isn't used at the first time frame. So you could write instead `cold(  '--c|'`... (two spaces) because that's when the cold observable would be subscribed to.